frontend/components/events/create/forms/BasicForm.tsx

```diff
@@ -177,7 +195,7 @@ export function BasicInformation({
               <Option value="volleyball">Volleyball</Option>
               <Option value="badminton">Badminton</Option>
               <Option value="basketball">Basketball</Option>
-              <Option value="soccer">Soccer</Option>
+              <Option value="Ssoccer">Soccer</Option>
```

Comment:
I presume this typo was fixed already? I can't see this in the branch, but not sure why it's still here.

Reply:
yep fixed